tests/10apidoc/09synced.pl

```diff
@@ -205,7 +205,34 @@ sub await_sync_presence_contains {
 }
 
 
-push @EXPORT, qw( assert_room_members assert_state_room_members_matches );
+push @EXPORT, qw( assert_room_members assert_state_room_members_matches assert_room_state);
+
+# assert that the state body of a sync response is made up of the given state types.
+sub assert_room_state {
```

Comment:
this should probably be `assert_room_state_types_match` or something, because (a) it only checks for the types, not the whole state, and (b) it checks that they match exactly (as opposed to, for example, being a superset of the desired list)

Reply:
Also, should it be `assert_state_types_match` and take a state block, ie pull lines 214-215 up to the caller, making this equivalent to `assert_state_room_members_matches` rather than `assert_room_members`? That feels like it would be more intuitive to me.